docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,70 @@
-# Updating the kit
+# Updating to the latest version
 
-**Important note**
+You can update to the latest version of the GOV.UK Prototype Kit if you want to either:
 
-If you have made any changes outside the `app` folder, this process will destroy those changes. We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside `app`, and add them back after updating.
+- use a new component
+- improve how your prototype meets accessibility requirements
 
-## Steps
+You can update using either:
 
-[Download the latest Prototype Kit](/docs/download) and unzip it.
+- Node.js package manager (npm)
+- a zip file - if updating with npm does not work 
 
-In your project, delete everything apart from the `app` and `.git` folder.
+## Update using npm
 
-Copy everything from the latest kit to your project, apart from the `app` folder.
+1. In your [terminal](https://govuk-prototype-kit.herokuapp.com/docs/install/requirements.md#terminal), `cd` to your prototype folder.
 
-Copy the config.js file from the `app` folder in the latest kit to the `app` folder of your prototype. If you've made any changes to the config.js file in your prototype then you'll need to re-enter them in the new version of the file e.g. the service name.
+2. Run `npm update govuk-frontend`.
 
-Check `\app\assets\sass\patterns` in the latest kit for any new patterns. Copy the files over to your prototype.
+3. [Run the kit and check it works](/docs/install/run-the-kit).
 
-Check `\app\assets\sass\application.scss` in the latest kit to see if any changes have been made in the top section, above where it says `// Add extra styles here`. Copy anything new from that file to the version in your prototype, making sure you don't overwrite any extra styles you have added yourself.
+If your prototype does not work or does not look right, try updating using a zip file instead.
 
-Check the [release notes for the latest release](https://github.com/alphagov/govuk-prototype-kit/releases/latest), for any additional changes you need to make to your prototype.
+## Update using a zip file
 
----
+1. Make a backup copy of all your prototype's folders.
```

Comment:
I'd say 'your prototype folder'

Reply:
Discussed with Joe and resolved in e87063a